azurerm/data_source_redis_cache_test.go

```diff
@@ -0,0 +1,74 @@
+package azurerm
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"testing"
+)
+
+func TestAccDataSourceAzureRMRedisCache_standard(t *testing.T) {
+	dataSourceName := "data.azurerm_redis_cache.test"
+	ri := tf.AccRandTimeInt()
+
+	name := fmt.Sprintf("acctestrediscache-%d", ri)
+	resourceGroupName := fmt.Sprintf("acctestRG-%d", ri)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMRedisCacheDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAzureRMRedisCache_standard(ri, testLocation()),
+			},
```

Comment:
we should be able to remove this step, since the Redis Cluster Resource is required for the Data Source